### DISCUSSION TITLE
vkCreateInstance failed -9 🚨 Error: Invalid GPU Device

### DISCUSSION THREAD

**Discussion by moindatori** [upscayl/upscayl#819] [Q&A]
hi i'm usning gtx 1660 super and have this issu can somone help me to solve this prob
![error](https://github.com/upscayl/upscayl/assets/132421039/5af182b8-c7b2-4ebe-8497-04dff88df986)

**Comment by NayamAmarshe** [comment]
Did you press troubleshoot?

**Reply by sellvebd** [reply]
Hi NayamAmarshe
I have faced the same issue in my computer. I presses troubleshoot and its open this link here https://docs.upscayl.org/introduction 
What is minimum supported Graphics card? My Graphics card model -- Asus Geforce Gt 730 2GB GDDR5 Graphics Card, Please give me a solution. Thanks in advanced

**Reply by aaronliu0130** [reply]
Check the troubleshooting link
